### ISSUE TITLE
Exactly stop loss and minimal roi

### ISSUE THREAD

**Issue by shotuhhh** [freqtrade/freqtrade#9537]
Hello,
Please help me, how I can set Stoploss and minimal_roi for workning Stoploss and minimal roi in exactly price or below that % I wrote before ?

minimal_roi = {
        "0": 0.01,
        "2": 0.018
    }

stoploss = -0.0255

I tried many type of command but nothing For ex I set stop loss lr minimal but it's work in higher % that I set.
Please say how I can exactly % I set before.

Best regards

**Comment by xmatthias** [comment]
Please get used to filling out the issue template.  Those who frequently omit the template may have their issues closed without comment.
We're asking some very basic questions - by deleting the issue template, you're explicitly choosing not to answer our questions.

Filling out the template helps reduce our workload, and we greatly appreciate that.

---


I'm not entirely sure what exactly you're asking (hence a bold guess)
If it's about the order fill prices (a stoploss may not fill at exactly -2.55% loss) - then this a simple fact of markets.

You trade 100% fills (market orders, which will have slippage) over "not everything fills if market moves too fast" (by using limit orders).
There's nothing around this - and you'll not get "exact" fills - not unless you accept that orders may not fill - which in case of a stoploss, can have damatic consequences.

**Comment by shotuhhh** [comment]
Ok thanks now I just add all info in template

**Comment by xmatthias** [comment]
Now a duplicate of #9539
